front/app/api/initiative_statuses/types.ts

```diff
@@ -12,8 +12,7 @@ export type InitiativeStatusCode =
   | 'expired'
   | 'threshold_reached'
   | 'answered'
-  | 'ineligible'
-  | 'custom';
```

Comment:
It's not possible to configure a custom initiative status (at least in the FE) at the moment.